.eslintrc.js

```diff
@@ -1119,6 +1119,39 @@ module.exports = {
       // All files
       files: ['x-pack/plugins/enterprise_search/**/*.{ts,tsx}'],
       rules: {
+        'import/order': [
+          'error',
+          {
+            groups: ['unknown', ['builtin', 'external'], 'internal', 'parent', 'sibling', 'index'],
+            pathGroups: [
+              {
+                pattern:
+                  '{../../../../../../,../../../../../,../../../../,../../../,../../,../}{common/,*}__mocks__{*,/**}',
```

Comment:
Yes this is absolutely horrifying. No there is no way else to do this. You would think that `**/__mocks__/**` would work. It absolutely does not.

See: https://github.com/benmosher/eslint-plugin-import/issues/1632

So yeah, I guess let's hope we don't go 7 folders in deep from a __mocks__ file, or we're adding yet another another `,` of 7x `../`s :dead_inside:

Reply:
Y'all are also free to test this yourselves (glob tester links are in the PR description, under resources) but I can also reassure you I spent several hours of my life yesterday and today trying to get this pathGroup work and this was the only way it did.